resources/views/pages/game/[game]/hashes/manage.blade.php

```diff
@@ -1,14 +1,25 @@
-@props([
-    'game',
-    'me',
-])
-
 <?php
+
 use App\Community\Enums\ArticleType;
+use App\Models\Game;
+use App\Models\GameHash;
+use function Laravel\Folio\{middleware, name};
+
+middleware(['auth', 'can:view,game', 'can:manage,' . GameHash::class]);
+name('game.hash.manage');
+
+?>
+
+@php
```

Comment:
It's a convention we have to follow that comes with Folio; all files in `pages` are auto-loaded and registered as routes (if it wasn't defined in a RouteServiceProvider yet).
While auto-loading it's executing each `<?php ?>` block in the files so it can extract additional infos about the route (name, middlewares). To prevent a php block to be executed during auto-loading phase `@php` can be used which is only executed at render time.

I tried to follow this convention in all the files that were moved so far.
It might take some time to get used to it, I know it will for me.

Reply:
Another change related to that:

`use` statements should always be at the very top, even in blade files.
I moved `use` statements and Folio-specific stuff to the topmost `<?php ?>` block and left everything else that is related to contents in `@php @endphp` blocks.